packages/data/addon/models/metadata/column.ts

```diff
@@ -110,4 +117,65 @@ export default class ColumnMetadataModel extends EmberObject implements ColumnMe
   tags?: string[];
 
   partialData?: boolean;
+
+  /**
+   * @property {string} columnFunctionId
+   */
+  columnFunctionId!: string;
+
+  /**
+   * Many to One relationship
+   * @property {ColumnFunction} columnFunction
+   */
+  get columnFunction(): ColumnFunction | undefined {
+    const { columnFunctionId, source, keg } = this;
+
+    if (columnFunctionId) {
+      const columnFunction = keg.getById('metadata/column-function', columnFunctionId, source);
+      return (columnFunction as unknown) as ColumnFunction;
+    }
+    return undefined;
+  }
+
+  /**
+   * @property {boolean} hasParameters
+   */
+  get hasParameters(): boolean {
+    return !!this.arguments?.length;
+  }
+
+  /**
+   * @property {object[]} arguments - arguments for the metric
+   */
+  get arguments(): TODO[] {
```

Comment:
```suggestion
  get arguments(): FunctionArgument[] {
```
via https://github.com/yahoo/navi/blob/request-v2/packages/data/addon/models/metadata/function-argument.ts